### DISCUSSION TITLE
Youtube block third party apps

### DISCUSSION THREAD

**Discussion by Raffy-ID** [libre-tube/LibreTube#5919] [General]
### Steps to reproduce

-

### Expected behavior

-

### Actual behavior

-

### LibreTube version

-

### Android version

-

### Other details

https://www.google.com/url?q=https://www.theverge.com/2024/4/15/24131338/youtube-ad-blocker-crackdown-mobile-apps&sa=U&ved=2ahUKEwjNopGn0saFAxU5b2wGHb30CJAQxfQBegQIBhAC&usg=AOvVaw3rvA-3CaGVvHfJaOORxira

Youtube has tried to remove ads blocker and third party apps from a new policy by Youtube. Starting from April 2024, third party apps must comply to youtube API. How this is affect Libretube?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
More buffering mostly, as Piped uses YouTube's internal API there's probably not much they could do about it other than ratelimiting and throttling instances.
